python/python/glide/async_commands/core.py

```diff
@@ -1100,6 +1100,25 @@ async def ttl(self, key: str) -> int:
         """
         return cast(int, await self._execute_command(RequestType.TTL, [key]))
 
+    async def type(self, key: str) -> str:
+        """
+        Returns the string representation of the type of the value stored at `key`.
+
+        See https://redis.io/commands/type/ for more details.
+
+        Args:
+            key (str): The key to check its data type.
+
+        Returns:
+            Optional[str]: The type of value stored at `key`, or "none" when key doesn't exist.
```

Comment:
nit: "none" => None

Reply:
ohh, I see, it doesn't response with None, but with a "none" string? if so, please change the return type documented here from Optional[str] to str

Reply:
Lets write it a bit clearer:
```
If the key exists, the type of the stored value is returned; otherwise, a "none" string is returned.